### PR TITLE
Improves a bit documentation for newbies like me

### PR DESCRIPTION
**Pull request by JJ**
I'm rather new to Go, so here are the things I needed to do which might help other newbies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,15 +60,15 @@ package main
 import (
     "log"
     "time"
-
+	"os"
```
Comment (tucnak) [inline README.md]: Please keep it consistent with the other imports

File: README.md [hunk 2]
```diff
@@ -46,15 +60,15 @@ package main
...
     "github.com/tucnak/telebot"
 )
 
 var bot *telebot.Bot
 
 func main() {
     var err error
-    bot, err = telebot.NewBot("SECRET TOKEN")
+    bot, err = telebot.NewBot(os.Getenv("BOT_TOKEN"))
```
Comment (tucnak) [inline README.md]: 👍 Totally makes sense, now I wonder why I didn't come up with this earlier.

File: README.md [hunk 3]
```diff
@@ -37,7 +46,12 @@ func main() {
 ## Inline mode
 
 As of January 4, 2016, Telegram added inline mode support for bots.
-Telebot support inline mode in a fancy manner.
+Telebot support inline mode in a fancy manner. Since you are probably
```
Comment (tucnak) [inline README.md]: I can't see how one is coming out of another. It all started with inline modes and there goes the env var thing?

File: README.md [hunk 4]
```diff
@@ -6,7 +6,16 @@
 
 Bots are special Telegram accounts designed to handle messages automatically. Users can interact with bots by sending them command messages in private or group chats. These accounts serve as an interface for code running somewhere on your server.
 
-Telebot offers a convenient wrapper to Bots API, so you shouldn't even care about networking at all. Here is an example "helloworld" bot, written with telebot:
+Telebot offers a convenient wrapper to Bots API, so you shouldn't even
+care about networking at all. Install it with
```
Comment (tucnak) [inline README.md]: I'd say "You may install it with..", but that's a nitpick.

**Comment by JJ** [comment]
Back to work. Thanks for the feedback!

File: README.md (hunk 3 above)
Comment (JJ) [inline README.md]: That's the actual program I worked with. Maybe I should have moved it up to the first example...

File: README.md (hunk 1 above)
Comment (JJ) [inline README.md]: It''s the spaces vs. tabs war all over again.. I'll fix that.

**Comment by JJ** [comment]
Thanks! Great job, BTW.

**Pull request merged**